script/tool/lib/src/common.dart

```diff
@@ -140,6 +144,13 @@ bool isLinuxPlugin(FileSystemEntity entity, FileSystem fileSystem) {
   return pluginSupportsPlatform(kLinux, entity, fileSystem);
 }
 
+/// Throws a [ToolExit] with `exitCode` and log the `errorMessage` in red.
+void PrintErrorAndExit({@required String errorMessage, int exitCode = 1}) {
```

Comment:
Why is this `Print` rather than `print`?

Reply:
Haha, it is probably caused by context switching between CPP and dart. Will fix